ui/src/common/PlayButton.js

```diff
@@ -46,8 +48,7 @@ export const PlayButton = ({ record, color, size, ...rest }) => {
         playAlbum(record)
       }}
       aria-label="play"
-      className={classes.icon}
-      {...rest}
+      className={theme === 'Light' ? classes.icon : rest.className}
```

Comment:
We don't want to check if the light theme is selected. What if we introduce another theme based on "Light", but with a different name?

The solution is to set the default color of the button to white, but it can be overridden by a any theme using the `NDAlbumGridView.albumPlayButton` class (as in the _Spotify-ish_ theme)

Reply:
I understand. I removed a lot of unnecesary code between `AlbumGridView.js` and `PlayButton.js` and included `albumPlayButton: { color: 'white' }` instead which fixes the issue at hand.